tools/tools/vt/mkkfont/mkkfont.c

```diff
@@ -96,7 +96,7 @@ print_mappings(struct font_header *fh, int map_index)
 		printf("{ 0x%04x, 0x%04x, 0x%02x },",
 		    be32toh(fm.vfm_src), be16toh(fm.vfm_dst),
 		    be16toh(fm.vfm_len));
-		col = (col + 1) % 2;
+		col ^= 1;
```

Comment:
This isn't equivalent. It will *only work* if col value is 1 or 0, anything else just will not. I don't think there would be a significant performance change with it.